ios/Converse/AppDelegate.mm

```diff
@@ -13,6 +16,22 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   // They will be passed down to the ViewController used by React Native.
   self.initialProps = @{};
 
+  [RNFBAppCheckModule sharedInstance];
+
+//  [FIRApp configure];
+  
+  
+  // todo - check environment variables from expo/react native and
```

Comment:
👀

Reply:
I'm not sure if any of this is expected to change, would need to verify with firebase docs

Reply:
@alexrisch this is here to dynamically select a `Google-Services Info.plist` based on environment

Reply:
forgoing this approach in favor of copying plist files for google in build/preview and build/production scripts